waggle-dance-core/src/main/java/com/hotels/bdp/waggledance/client/pool/MetaStoreClientPool.java

```diff
@@ -31,7 +31,7 @@
  */
 public class MetaStoreClientPool implements KeyedObjectPool<AbstractMetaStore, CloseableThriftHiveMetastoreIface> {
 
-  private final static Logger log = LoggerFactory.getLogger(MetaStoreClientPool.class);
+  private final static Logger LOG = LoggerFactory.getLogger(MetaStoreClientPool.class);
```

Comment:
OMG, going back the other way now! ;P

Reply:
Don't MIX and match I guess, I gave up :)